### ISSUE TITLE
Feature request: Question feedback images

### ISSUE THREAD

**Issue by guywillis** [adaptlearning/adapt_framework#2390]
Feature request to support images, with alt text if required, in question feedback. Images should be able to mirror the question feedback text so there are correct, partly correct not / final, and incorrect not final versions.

If an image is defined, the layout of the text and image is split 60% / 40% in favour of the text. If no image is defined the text defaults to 100% width.

Schemas will also need to be updated.

Suggestion:

```
"_feedback": {
  "_title": "Title",
  "_correct": {
    "body": "Correct feedback text",
    "_graphic": {
      "alt": "Alt text",
      "_src": "course/en/images/correct.png",
    }
  },
  "_partlyCorrect": {
    "_notFinal": {
      "body": "Partly correct not final feedback text",
      "_graphic": {
        "alt": "Alt text",
        "_src": "course/en/images/partly-correct-not-final.png"
      }
    },
    "_final": {
      "body": "Partly correct final feedback text",
      "_graphic": {
        "alt": "Alt text",
        "_src": "course/en/images/partly-correct-final.png"
      }
    }
  },
  "_incorrect": {
    "_notFinal": {
      "body": "Incorrect not final feedback text",
      "_graphic": {
        "alt": "Alt text",
        "_src": "course/en/images/incorrect-not-final.png"
      }
    },
    "_final": {
      "body": "Incorrect final feedback text",
      "_graphic": {
        "alt": "Alt text",
        "_src": "course/en/images/incorrect-final.png"
      }
    }
  }
}
```

**Comment by moloko** [comment]
do you also want to go for option-specific feedback images for those question components that support that functionality (MCQ & GCMQ)?

we could always leave for now and add if and when someone says they definitely need it

**Comment by guywillis** [comment]
Ideally, yes. Get it all done in one fell swoop.

**Comment by guywillis** [comment]
I had a stab a little while ago at the request in the enhancement/refactor branch in the Framework, MCQ, and Tutor plugins which could form the basis for the request.

Framework: https://github.com/adaptlearning/adapt_framework/commit/181c13ed0da323454f5771de6b53dbbefa587b61

MCQ:
https://github.com/adaptlearning/adapt-contrib-mcq/commit/175e4d55eb084bd1c436edbe91793bdc69e41e4f

Tutor:
https://github.com/adaptlearning/adapt-contrib-tutor/commit/9c838b326fdb3dcf19dc1784ac24adf6d3199990

**Comment by guywillis** [comment]
Updated original request to include generic feedback title + ability to override title per feedback state.

**Comment by oliverfoster** [comment]
@guywillis is this still relevant?

**Comment by guywillis** [comment]
The method described above to implement, maybe not. The underlying request, absolutely.

**Comment by oliverfoster** [comment]
Thanks

**Comment by oliverfoster** [comment]
add this to requirements https://github.com/adaptlearning/adapt_framework/issues/2408
